app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -168,25 +152,19 @@ def update_invalid_user
     current_user.increment_second_factor_attempts_count!
   end
 
-  def handle_valid_verification_for_confirmation_context
-    user_session[:authn_at] = Time.zone.now
-    track_mfa_method_added
-    @next_mfa_setup_path = next_setup_path
```

Comment:
Noticing we don't always retain this `@next_mfa_setup_path` except in OtpVerificationController, but it's referenced in the shared `after_otp_verification_confirmation_url` method. Do we need it for other MFA methods?